docs/guides/prompt-manager/overview.mdx

```diff
@@ -1,73 +1,55 @@
 ---
-title: Overview Prompt Editor 
+title: Prompt Editor
 description: Learn about the powerful features of the Latitude prompt editor.
 ---
 
-## Prompt Editor
-The Latitude prompt editor allows you to define and structure your prompts with precision.
-
-## Model Configuration
-At the top of the prompt editor you can specify the default provider and AI model to use in your prompt. You can modify it using the input or directly in the code. 
-
-To add a provider, you can check the [Providers](/guides/getting-started/providers) page.
+## Overview
 
-For example:
-
-```
----
-provider: openai
-model: gpt-4
-temperature: 0.7
----
-```
-You can check [Prompt config section](/guides/prompt-manager/parameters) to learn more details.
+The Latitude prompt editor allows you to define and structure your prompts with precision.
 
-## Defining Conversation Structure
+![Prompt Editor](/assets/prompt_editor.png)
 
-The Latitude prompt editor uses different roles to structure conversations and define the behavior of the AI model. The main roles include:
+The editor allows to define prompts using [PromptL](/promptl/getting-started/introduction), a simple and intuitive language that enables you to create dynamic yet easy-to-understand prompts.
 
-- [**System**](/guides/prompt-manager/roles#system-role): Sets the overall context and behavior for the AI model.
-- [**User**](/guides/prompt-manager/roles#user-role): Represents the input or questions from the end-user.
-- [**Assistant**](/guides/prompt-manager/roles#assistant-role): Represents the AI model's responses.
+## Syntax
 
-To learn more about how to use these roles effectively in your prompts, please visit our [Roles in Prompt Editor](/guides/prompt-manager/roles) page.
+The language syntax is easy to understand and use, making it simple to create dynamic and engaging prompts. However, it also supports advanced features like parameters, snippets, and logic to create more complex and dynamic prompts.
 
+To learn more about it, take a look at the [PromptL documentation](/promptl/getting-started/introduction):
 
-### Advanced features
-The Latitude prompt editor also offers advanced features to create more dynamic and complex prompts:
+- [Structure](/promptl/syntax/structure)
+- [Messages](/promptl/syntax/messages)
+- [Configuration](/guides/prompt-manager/configuration)
+- [Variables](/promptl/syntax/variables)
+- [Conditional statements](/promptl/syntax/conditionals)
+- [Loops and Iteration](/promptl/syntax/loops)
+- [Snippets and references](/promptl/advanced/snippets)
+- [Chains and Steps](/promptl/advanced/chains)
```

Comment:
Most of the documentation is just pointing to PromptL docs.

(Except for `Congifuration`, which has its own page to list all the available configurations supported by vercel)